app/javascript/stylesheets/bootstrapOverrides.scss

```diff
@@ -0,0 +1,20 @@
+.btn-primary {
```

Comment:
Much of this is due to my changing the value of `$primary`, which I did to change the color of the SUL footer links. I suspect I did exactly the opposite of what I should have done, which is to add a new class to these links to force them to be Stanford digital blue instead of the BS primary color (Stanford digital red). Maybe there's a bootstrappier way to do this. Open to ideas.

Reply:
I think you're right. I think we want a style for 
```scss
footer {
  a {
    color: $stanford-digital-blue;
  }
}
```

Reply:
Done.